openslides_backend/action/actions/meeting/import_.py

```diff
@@ -631,6 +629,26 @@ def append_extra_events(
                 )
             )
 
+        # add new users to the organization.user_ids
+        new_user_ids = []
+        for user_entry in json_data.get("user", {}).values():
```

Comment:
Are you sure, that the user_ids from json_data are the new ones or still the old ones from export?

Reply:
I except the json_data["user"] to be merged ones and new ones. And then use only the new ones.

Reply:
Perhaps here is an error. I will look at it again.

Reply:
Ah, there was a bug, I changed the code.